### PR TITLE
Add new graphql to get customer by name

### PR DESCRIPTION
**Pull request by igayoso**
Create new graphql query to get customer by name.

### CODE REVIEW

File: services/api/src/schema.js [hunk 1]
```diff
@@ -93,6 +93,7 @@ const typeDefs = `
 
   type Query {
     projectByName(name: String!): Project
+    customerByName(name: String!): Project
```
Comment (Schnitzel) [inline services/api/src/schema.js]: should be `Customer` no `Project`

**Review by Schnitzel — CHANGES_REQUESTED** [review]
just one tiny thing

File: services/api/src/schema.js (hunk 1 above)
Comment (igayoso) [reply services/api/src/schema.js]: 👍

**Review by Schnitzel — APPROVED** [review]
lets see what the CI says :)

**Comment by ryyppy** [comment]
I will merge this in our #239 PR

**Comment by ryyppy** [comment]
Added to #239 .. see https://github.com/amazeeio/lagoon/pull/239/commits/c91c422f80f66bbdd1f0d728de17508b2c60ed64

This can be closed now!

**Pull request closed without merging**